tests/test_database.py

```diff
@@ -547,24 +547,24 @@ def test_motohawk_encode_decode(self):
                     'AverageRadius': 0,
                     'Enable': 0
                 },
-                b'\x01\x00\x20\x00\x00\x00\x00\x00'
+                b'\x01\x00\x40\x00\x00\x00\x00\x00'
```

Comment:
any idea what the reason for this is?

Reply:
It's unnecessary with rounding. I reversed the changes